todo_app/app.py

```diff
@@ -1,6 +1,6 @@
 from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
-from todo_app.data.trello_items import get_trello_items, add_trello_item, update_item_status
+from todo_app.data.db_items import get_todo_items, add_todo_item, update_item_status
```

Comment:
Nice, the fact this file has such limited changes is a really strong sign that you've designed your interfaces well so other parts of the code aren't impacted too much by the underlying changes 😄